src/app/frontend/common/components/middleellipsis/component.js

```diff
@@ -19,10 +19,26 @@ export default class MiddleEllipsisController {
   /**
    * Constructs middle ellipsis controller.
    * @ngInject
+   * @param {!angular.JQLite} $element
    */
-  constructor() {
+  constructor($element) {
     /** @export {string} Initialized from the scope. */
     this.displayString;
+
+    /** @private {!angular.JQLite} */
+    this.element_ = $element;
+  }
+
+  isTrancated() {
```

Comment:
Missing annotations. Check our Wiki pages for more info about code conventions.

Reply:
Which annotation is mistaken?(this.element_ or isTrancated or any other else ?)
this.element_ is used in  MiddleEllipsisController only, and isTruncated is used in middleellipsis.html.

Reply:
Closure compiler annotations - read about them in our wiki.

Reply:
My added annotations( @ param {!angular.JQLite} $element and @ private {!angular.JQLite}) will same as other components such as (common/components/allocatedresourceschart/component.js)
I'm sorry, but could you tell me about a detail of mistaken annotetions.